camel-xml-jaxb is missing 'org.apache.camel.model.app' Import

`Caused by: jakarta.xml.bind.JAXBException: "org.apache.camel.model.app" doesnt contain ObjectFactory.class or jaxb.index
	at org.glassfish.jaxb.runtime.v2.ContextFactory.createContext(ContextFactory.java:241) ~[!/:4.0.5 - cb19596]
	at org.glassfish.jaxb.runtime.v2.JAXBContextFactory.createContext(JAXBContextFactory.java:58) ~[!/:4.0.5 - cb19596]
	at jakarta.xml.bind.ContextFinder.find(ContextFinder.java:322) ~[!/:4.0.2]
	at jakarta.xml.bind.JAXBContext.newInstance(JAXBContext.java:392) ~[!/:4.0.2]
	at jakarta.xml.bind.JAXBContext.newInstance(JAXBContext.java:349) ~[!/:4.0.2]
	at org.apache.camel.xml.jaxb.DefaultModelJAXBContextFactory.newJAXBContext(DefaultModelJAXBContextFactory.java:44) ~[!/:?]
	at org.apache.camel.xml.jaxb.DefaultModelJAXBContextFactory.newJAXBContext(DefaultModelJAXBContextFactory.java:32) ~[!/:?]
	at org.apache.camel.xml.jaxb.JaxbHelper.getJAXBContext(JaxbHelper.java:78) ~[!/:?]
	at org.apache.camel.xml.jaxb.JaxbHelper.loadRoutesDefinition(JaxbHelper.java:389) ~[!/:?]`

Class DefaultModelJAXBContextFactory is providing its class loader which does not "see" the org.apache.camel.model.app package.

--- a/core/camel-xml-jaxb/pom.xml
+++ b/core/camel-xml-jaxb/pom.xml
@@ -37,6 +37,14 @@
             org.apache.camel*;version=${camel-version}
         </camel.osgi.export>
         <camel.osgi.import>
+            org.glassfish.jaxb.runtime.v2;resolution:=optional,
+            org.apache.camel.model.app,
+            org.apache.camel.model.cloud,
+            org.apache.camel.model.config,
+            org.apache.camel.model.errorhandler,
+            org.apache.camel.model.loadbalancer,
+            org.apache.camel.model.transformer,
+            org.apache.camel.model.validator,
             *
         </camel.osgi.import>
     </properties>

